packages/js/src/elementor/components/fills/ElementorFill.js

```diff
@@ -50,75 +51,77 @@ export default function ElementorFill( { isLoading, onLoad, settings } ) {
 	}
 
 	const webinarIntroElementorUrl = get( window, "wpseoScriptData.webinarIntroElementorUrl", "https://yoa.st/webinar-intro-elementor" );
-
+	const locationContext = { locationContext: "elementor-sidebar" };
 	return (
 		<>
 			{ isWordProofIntegrationActive() && <WordProofAuthenticationModals /> }
 			<Fill name="YoastElementor">
-				<SidebarItem renderPriority={ 1 }>
-					<Alert />
-					<WebinarPromoNotification hasIcon={ false } image={ null } url={ webinarIntroElementorUrl } />
-				</SidebarItem>
-				{ settings.isKeywordAnalysisActive && <SidebarItem renderPriority={ 8 }>
-					<KeywordInput
-						isSEMrushIntegrationActive={ settings.isSEMrushIntegrationActive }
-					/>
-					{ ! window.wpseoScriptData.metabox.isPremium && <Fill name="YoastRelatedKeyphrases">
-						<SEMrushRelatedKeyphrases />
-					</Fill> }
-				</SidebarItem> }
-				<SidebarItem renderPriority={ 25 }>
-					<GooglePreviewModal />
-				</SidebarItem>
-				{ settings.displayFacebook && <SidebarItem renderPriority={ 26 }>
-					<FacebookPreviewModal />
-				</SidebarItem> }
-				{ settings.displayTwitter && <SidebarItem renderPriority={ 27 }>
-					<TwitterPreviewModal />
-				</SidebarItem> }
-				{ settings.displaySchemaSettings && <SidebarItem renderPriority={ 28 }>
-					<SidebarCollapsible
-						title={ __( "Schema", "wordpress-seo" ) }
-					>
-						<SchemaTabContainer />
-					</SidebarCollapsible>
-				</SidebarItem> }
-				{ settings.displayAdvancedTab && <SidebarItem renderPriority={ 29 }>
-					<SidebarCollapsible
-						title={ __( "Advanced", "wordpress-seo" ) }
-					>
-						<AdvancedSettings location="sidebar" />
-					</SidebarCollapsible>
-				</SidebarItem> }
-				{ settings.isKeywordAnalysisActive && <SidebarItem renderPriority={ 10 }>
-					<Fragment>
-						<SeoAnalysis
+				<Root context={ locationContext }>
```

Comment:
Is there a reason you went with the Fill for Elementor instead of outside of it like you did for Classic and Block?
I.e. you could render the root higher up in the tree, here: https://github.com/Yoast/wordpress-seo/blob/trunk/packages/js/src/initializers/elementor-editor-integration.js#L167